src/Xamarin.Android.App/Xamarin.Android.App.Runtime.proj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.88">
+
```

Comment:
I'm feeling as the number of MSBuild files start to grow...

@dellis1972 do you think we should start putting a short comment/description at the top of each new MSBuild file to explain what the file is?

So we would want just a short sentence explaining the difference between `App.Runtime.proj` and `App.Ref.proj`? `Xamarin.Android.Lite.proj`, too?

If I made a `Xamarin.Android.BuildOrder.targets` the name is almost self-explanatory, but would probably be helpful to say "this is where all `$(FooDependsOn)` properties go".

Reply:
Alternatively in this case, we could use a single project/targets file that could create multiple NuGet packages depending on which target is called? These projects don't have any sources and are only here to try to simplify `dotnet pack` invocations.

Reply:
I think it's good to have separate files; it gives an opportunity to have a comment that explains what each `.nupkg` file is.

Reply:
comments are good 👍